src/unxt/_src/units/system/core.py

```diff
@@ -213,6 +220,51 @@ def unitsystem(flag: type[StandardUnitSystemFlag], *units_: Any) -> AbstractUnit
     return unitsystem(*units_)
 
 
+@dispatch  # type: ignore[no-redef]
+def unitsystem(
+    flag: type[DynamicalSimulationUnitSystemFlag],
+    *units_: Any,
+    G: u.Quantity | float | int = 1.0,  # noqa: N803
```

Comment:
I agree it's probably good to start with just float/int. We can add quantity later if necessary.